.azure-pipelines/build-plcrashreporter-1ES.yml

```diff
@@ -1,137 +1,72 @@
 trigger:
-- master
+  - "master"
 pr:
-- master
+  - "master"
 
 variables:
-  Configuration: Release
-  SDK: ''
-  XCODE_PATH: '/Applications/Xcode_14.3.1.app/Contents/Developer'
+  Configuration: "Release"
+  SDK: ""
```

Comment:
I can't find if we use this variable. Can you check if it's used, if not lets remove it.

Reply:
It's required by Xcode build.